README.md

```diff
@@ -26,30 +28,45 @@ saleor-sdk tools decode-id VXNlcjoyMg==
 saleor-sdk tools encode-id User 22
 ```
 
-## Documentation
+## Development
+
+To contribute to this repository you will need Hatch to setup a local development environment. 
+
+Install [Hatch](https://hatch.pypa.io/latest/install/#pipx).
+
+### Documentation
+
+You can deploy a local documentation service, it reloads changes and allow for a live preview of how the documentation will look like after publication:
 
 ```
 hatch run docs:serve
 ```
 
 and navigate to http://127.0.0.1:8000
 
-## Tests
+### Tests
+
+To run tests suite use the following command: 
 
 ```
 hatch run test 
 ```
 
-## Build and deploy
+### Code style and linters
+
+Use the following commands to format the code and lint it for issues:
 
 ```
-hatch build -c
-hatch publish
+hatch run lint:fmt
+hatch run lint:all
 ```
 
-## Code style
+
+### Build and deploy
+
+This is done by a CI/CD workflow upon the creation of a release but in case of the need for a manual publication of the packager to PyPI you need to build the package archive and publish it. 
```

Comment:
```suggestion
To publish a new version to PyPI, update it's version number in the `pyproject.toml` file and create new github release.

In case when you need to make a new release without GitHub workflow, use following `hatch` commands:
```